src/components/SignUp/GeneralInfo.js

```diff
@@ -7,13 +7,13 @@ import {
   ScrollView,
   StyleSheet,
   TextInput,
-  TouchableWithoutFeedback,
   TouchableOpacity,
   View,
 } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
 import EditForm from './EditForm';
 import EduForm from './EduForm';
-import LinearGradient from 'react-native-linear-gradient';
+import EduFormIOS from './EduFormIOS';
```

Comment:
Because our application uses picker component.
Android picker has a modal but iOS has not modal. So, I made a modal for iOS picker.